lib/src/index.node.ts

```diff
@@ -0,0 +1,13 @@
+import { NanoTDFClient, NanoTDFDatasetClient, AuthProviders, version, clientType } from './index';
+import fetch from 'node-fetch';
+// @ts-ignore
+import crypto from 'crypto';
+
+// @ts-ignore
+globalThis.crypto = crypto;
```

Comment:
Is this safe? I was worried about polluting globals so I avoided it in my node PR.

I didn't bother with fetch though :-/